modules/silauth/src/Auth/Source/auth/IdBroker.php

```diff
@@ -95,16 +95,4 @@ public function getAuthenticatedUser(string $username, string $password): ?array
             $userInfo['member'] ?? []
         );
     }
-
-    /**
-     * Ping the /site/status URL. If the ID Broker's status is fine, the
-     * response string is returned. If not, an exception is thrown.
-     *
-     * @return string "OK"
-     * @throws Exception
-     */
-    public function getSiteStatus(): string
```

Comment:
Why is this being removed? Is there no more pinging?

Reply:
Yes, but since ID Broker has its own status check pinged directly by ECS and NodePing, we removed the check done from the ssp container a while back.

Reply:
What function does NodePing trigger?

Reply:
https://github.com/silinternational/ssp-base/blob/develop/modules/silauth/public/status.php